gtts/tts.py

```diff
@@ -373,3 +384,17 @@ def infer_msg(self, tts, rsp=None):
                 cause = "Uptream API error. Try again later."
 
         return "{}. Probable cause: {}".format(premise, cause)
+
+
+def _requests_proxies_arg(proxy) -> Optional[Dict[str, str]]:
+    """Returns a value suitable for the 'proxies' argument to 'requests.request."""
+    if proxy is None:
+        return None
+    elif isinstance(proxy, str):
+        return {"http": proxy, "https": proxy}
+    elif isinstance(proxy, dict):
+        return proxy.copy()
+    else:
+        raise ValueError(
+            "'proxy' must be specified as either a string URL or a dict with string URL under the https and/or http keys."
+        )
```

Comment:
proxy (dict, optional): A dictionary of proxy settings to use when
            connecting to Google Translate. For example::
            proxy = {   'http': '192.168.3.3:7890, 'https': '192.168.3.3:7893'}
            or:: proxy = '192.168.3.3:7890' ,call by _requests_proxies_arg(proxy) it will be converted to::
            proxy = {   'http': 'http://192.168.3.3:7890','https':'https://192.168.3.3:7890'}